contacts/index.js

```diff
@@ -52,8 +52,8 @@ route.get("/:id", authenticate, (req, res) => {
     });
 });
 
-route.delete("/:id", authenticate, (req, res) => {
-  const { id } = req.params;
+route.delete("/:id/:groupId", authenticate, (req, res) => {
```

Comment:
this works, but consider that you could also probably get the groupId by using the contactId and querying the database